components/molecules/Breadcrumbs.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Breadcrumb from './Breadcrumb';
+import BreadcrumbLink from './BreadcrumbLink';
+
+class Breadcrumbs extends Component {
+    static displayName = 'Breadcrumbs';
+
+    static defaultProps = {
+        linkComponent: BreadcrumbLink,
+        titleDecorator: title =>
+            title.replace(/^\w/, chr => chr.toUpperCase()).replace(/-|_/g, ' '),
+        hideMenus: false,
+        hideRoot: false
+    };
+
+    static propTypes = {
+        linkComponent: PropTypes.oneOfType([PropTypes.element, PropTypes.func]).isRequired,
+        config: PropTypes.shape({
+            title: PropTypes.string.isRequired,
+            path: PropTypes.string,
+            subpaths: PropTypes.array
+        }).isRequired,
+        pathname: PropTypes.string.isRequired,
+        titleDecorator: PropTypes.func,
+        hideMenus: PropTypes.bool,
+        hideRoot: PropTypes.bool
+    };
+
+    // Create the full path for the current slug based on the previous entry
+    _buildHref = (arr, slug) => {
+        const previous = arr && arr[arr.length - 1];
+        const newPath = previous ? `${previous.path}/${slug}` : slug;
+        return newPath.replace('//', '/');
+    };
+
+    // Create a title for parameters, either the capitalized param or a number
+    _getParamTitle = param =>
+        Number.isInteger(parseInt(param)) ? param : this.props.titleDecorator(param);
+
+    // Creates a flat Array only with the nodes that correspond to the current pathname
+    // It includes replaces the path with the full path to each link
+    _findTrail = (pathname, config) => {
+        const initialBreadcrumb = {
+            title: config.title || '/',
+            path: config.path.charAt(0) === '/' ? config.path : `/${config.path || ''}`
+        };
+
+        // Set initial data if necessary
+        const initialData = config.path === '/' ? [initialBreadcrumb] : [];
+        // Set index difference between breadcrumbs and path
+        const pathToBreadcrumbsDiff = initialData.length ? 2 : 1;
```

Comment:
i think the issue is here, it works with 1 : 0

Reply:
I didn't see this before, I ended up changing some of the logic a bit but it should be fixed now.